examples/gps_datalogging.py

```diff
@@ -51,7 +51,7 @@
 # uart = serial.Serial("/dev/ttyUSB0", baudrate=9600, timeout=10)
 
 # If using I2C, we'll create an I2C interface to talk to using default pins
-# i2c = busio.I2C(board.SCL, board.SDA)
+# i2c = board.I2C(board.SCL, board.SDA)
```

Comment:
these should all be `board.I2C()` without any arguments. `board.I2C()` uses the default pina.s

Reply:
Ah, ok. I'll fix that when I've got a sec